packages/controller/lib/setup/setupRepo.js

```diff
@@ -397,14 +409,21 @@ function Repo(options) {
                 }
 
                 // if we changed the name of the activeRepo, we should set newName as active repo
-                if (sysConfigObj && sysConfigObj.common && sysConfigObj.common.activeRepo === oldName) {
-                    sysConfigObj.common.activeRepo = newName;
+                if (sysConfigObj && sysConfigObj.common &&
+                    ((typeof sysConfigObj.common.activeRepo === 'string' && sysConfigObj.common.activeRepo === oldName) ||
+                     (typeof sysConfigObj.common.activeRepo === 'object' && sysConfigObj.common.activeRepo.includes(oldName)))
```

Comment:
If you want to make sure it is an array your should use Array.isArray(...) ... I would not use an object check ...